storage/database/doc.go

```diff
@@ -14,5 +14,28 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
 
-// Package database implements various types of databased used in Klaytn.
+/*
+Package database implements various types of databases used in Klaytn.
+This package is used to read/write data from/to the persistent layer.
+
+DBManager is the interface used by the consumers of database package.
```

Comment:
How about add one header for those explaination?

Reply:
Good point =) I will add one.